tests/test_main.py

```diff
@@ -178,7 +178,7 @@ def test_dog_bring_slippers_method():
     f = io.StringIO()
     with redirect_stdout(f):
         dog.bring_slippers()
-    assert f.getvalue() == "The slippers delivered!\n"
+    assert f.getvalue() == "The sleepers delivered!\n"
```

Comment:
why did you change it??
![image](https://github.com/mate-academy/py-feed-animals/assets/104974881/057207d4-3bdc-41bd-8752-e4d559d87dff)
![image](https://github.com/mate-academy/py-feed-animals/assets/104974881/fcb864ea-b5b8-430f-865c-fcb3559c44e2)